File: Electronics_Shop_17.Model/Requests/AddReview.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddReview
    {
        public int Rating { get; set; }
        public string Comment { get; set; }
        public int CustomerId { get; set; }
        public int ProductId { get; set; }
    }
}

File: angular17_web/src/app/models/review/add.review.ts
export class AddReview {
  rating!: number;
  comment!: string;
  customerId!: number;
  productId!: number;

  constructor(data?: any) {
    if (!data) return;

    this.rating = data.rating;
    this.comment = data.comment;
    this.customerId = data.customerId;
    this.productId = data.productId;
  }
}
